ob1k-core/src/test/java/com/outbrain/ob1k/client/http/BasicClientRpcTest.java

```diff
@@ -465,7 +466,7 @@ public String handleSync(final SyncClientRequestContext ctx) throws ExecutionExc
   @Test(expected=ExecutionException.class)
   public void testEmptyTargetBehavior() throws ExecutionException, InterruptedException {
     IHelloService service = new ClientBuilder<>(IHelloService.class).build();
-    final ComposableFuture<com.outbrain.ob1k.Response> future =  service.emptyString(); // used to throw "JsonMappingException: No content to map due to end-of-input"
+    final ComposableFuture<String> future =  service.helloWorld();
```

Comment:
Basically you changed the test to test something else here.
Why did you do that?